main.py

```
@@ -0,0 +1,56 @@
+with open('project_config.yml', 'r') as file:
+    config = yaml.safe_load(file)
+
+print(yaml.dump(config, default_flow_style=False))
+
+# COMMAND ----------
+
+# MAGIC %run  "/Users/chris.wroe@fabledata.com/marvelous-databricks-course-Chris-W-codes/Pre_processing"
+
+# COMMAND ----------
+
+preprocessor = PreProcess()
```

Ruff

Undefined name `PreProcess`

(F821)